### PR TITLE
Tweaking <code> styles to better fit PatternFly brand

### PR DESCRIPTION
**Pull request by rhamilto**
Fixes https://github.com/patternfly/patternfly/issues/254

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
LGTM

File: less/variables.less [hunk 1]
```diff
@@ -290,6 +290,7 @@
 @btn-primary-bg:                                                    #0085cf;
 @btn-primary-border:                                                #006e9c;
 @caret-width-base:                                                  0;
+@code-bg:                                                           #f5f5f5;
```
Comment (andresgalante) [inline less/variables.less]: Can you please reference this color variable?
https://github.com/patternfly/patternfly/blob/master/less/color-variables.less#L1

**Comment by rhamilto** [comment]
@andresgalante, I made the requested change so that @code-bg is now set to @color-pf-black-100.

**Comment by andresgalante** [comment]
👍 thanks @rhamilto. We are merging a PR from @cardosogabriel this week that normalize all colors using variables from the color pallet on color-variables. Moving forward we want to use just those colors.

**Comment by rhamilto** [comment]
Within the context of OpenShift, we've noticed this change poses a usability issue as now code sort of looks like a link.  See https://github.com/openshift/origin-web-console/pull/16#issuecomment-219562405.  Paging @kybaker and @LHinson.

**Comment by andresgalante** [comment]
@rhamilto thanks I'll check with @kybaker and @LHinson today

**Comment by LHinson** [comment]
@rhamilto totally see what you mean, def looks like a link. We'll take a look.

**Pull request merged**